### DISCUSSION TITLE
Number of CPUs

### DISCUSSION THREAD

**Discussion by CROprogrammer** [containers/podman#21222] [Q&A] [answered]
Hi,

when I set up my podman machine to use 1 CPU:

![Snimka zaslona 2024-01-10 161211](https://github.com/containers/podman/assets/73464772/07f023ee-0ff6-4a7f-a9a9-802edafa6fed)

it still defaults to 12 CPUs after I create it:

![Snimka zaslona 2024-01-10 162732](https://github.com/containers/podman/assets/73464772/2fd53ba9-372f-4d35-9356-c068114e02ba)

Is there solution for that?

**Comment by tnk4on** [comment, marked as answer]
The following is a transcription from "[Podman in Action](https://developers.redhat.com/e-books/podman-action)".
```
The physical limits specified in table F.2 (e.g., CPU, memory, and disk) are currently ignored on Windows, since the Windows Subsystem for Linux (WSL) backend dynamically resizes and shares resources across distributions. If you need to constrain resources, you can configure those limits in your users’ .wslconfig file. However, they apply globally to all WSL 2 distros, since they share the same underlying VM.
```

**Comment by afbjorklund** [comment]
Should be fixed by the Hyper-V provider, that starts a VM.
